plugins/devtools/src/components/Content/ExternalDependenciesContent/ExternalDependenciesContent.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Progress,
+  StatusError,
+  StatusOK,
+  StatusWarning,
+  Table,
+  TableColumn,
+} from '@backstage/core-components';
+import { ExternalDependency } from '@backstage/plugin-devtools-common';
+import {
+  Box,
+  createStyles,
+  Grid,
+  makeStyles,
+  Paper,
+  Theme,
+  Typography,
+} from '@material-ui/core';
+import { Alert } from '@material-ui/lab';
+import React from 'react';
+import { useExternalDependencies } from '../../../hooks';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    paperStyle: {
+      padding: theme.spacing(2),
+    },
+  }),
+);
+
+export const getExternalDependencyStatus = (
+  result: Partial<ExternalDependency> | undefined,
+) => {
+  switch (result?.status) {
```

Comment:
Can we not use the `ExternalDependencyStatus` enum here instead?

Reply:
Yes, for sure. Done! 🚀